webpack/ForemanInventoryUpload/Components/PageHeader/components/SyncButton/SyncButtonActions.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import API from 'foremanReact/API';
+import { addToast } from 'foremanReact/redux/actions/toasts';
+import { inventorySyncUrl } from './SyncButtonHelpers';
+import {
+  SYNCING_REQUEST,
+  SYNCING_SUCCESS,
+  SYNCING_FAILURE,
+} from './SyncButtonConstants';
+import Toast from './components/Toast';
+
+export const handleSync = () => async dispatch => {
+  dispatch({
+    type: SYNCING_REQUEST,
+    payload: {},
+  });
+  try {
+    const {
+      data: { syncHosts, disconnectHosts },
+    } = await API.post(inventorySyncUrl('tasks'));
+    dispatch({
+      type: SYNCING_SUCCESS,
+      payload: {
+        syncHosts,
+        disconnectHosts,
+      },
+    });
+
+    dispatch(
+      addToast({
+        sticky: true,
+        type: 'success',
+        title: 'Sync process ended successfully',
+        message: (
+          <Toast syncHosts={syncHosts} disconnectHosts={disconnectHosts} />
+        ),
+      })
+    );
```

Comment:
last commit is still in progress, but this is how the toast will showup after a successful sync:

![toast_sync](https://user-images.githubusercontent.com/26363699/88804447-96a6fa80-d1b6-11ea-8a0e-9e3b4432b68b.png)